lib/src/visitor/serialize.dart

```diff
@@ -19,6 +19,15 @@ import 'interface/css.dart';
 import 'interface/selector.dart';
 import 'interface/value.dart';
 
+enum LineFeed { CR, CRLF, LF, LFCR }
```

Comment:
Official Dart enums suck :stuck_out_tongue_closed_eyes:. If you use a custom class, you can just add the text as a field.

Also, I believe these should be all lowercase.